src/hb-ot-glyph.cc

```diff
@@ -129,6 +129,8 @@ struct _hb_ot_glyph_path_vectors
   {
     coords->fini ();
     commands->fini ();
+    free (coords);
+    free (commands);
```

Comment:
@khaledhosny, my bad, sadly :/ just wanted to ping you if you testing it somewhere, and there is some other issue on accessing tables that feels isn't limited to this work which currently makes tests fail, will try to file that as a separate but just wanted to mention the work may is very unstable :/